Unable to run `npm init directus-extension`; Inquirer no-default-import

### Describe the Bug

When executing `$ npm init directus-extension`, the command fails with a `SyntaxError`.

```sh
file:///home/cevans/.npm/_npx/9ea216cce9aeac9a/node_modules/create-directus-extension/lib/index.js:4
import { inquirer } from 'inquirer';
         ^^^^^^^^
SyntaxError: Named export 'inquirer' not found. The requested module 'inquirer' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'inquirer';
const { inquirer } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:190:5)

Node.js v18.13.0
```

### To Reproduce

1. Install Node.js 18.x
2. Run Command; `$ npm init directus-extension`
3. Observe Error

### Hosting Strategy

Self-Hosted (Custom)

Also unable to build extensions:

```sh
directus-extension-layout-revisions@1.0.0 build
> directus-extension build

node:internal/errors:491
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/path/to/apps/content-manager-pro/extensions/layouts/directus-extension-layout-revisions/node_modules/@directus/extensions-sdk/dist/esm/cli/run.js' imported from /path/to/apps/content-manager-pro/extensions/layouts/directus-extension-layout-revisions/node_modules/@directus/extensions-sdk/cli.js
    at new NodeError (node:internal/errors:400:5)
    at finalizeResolution (node:internal/modules/esm/resolve:326:11)
    at moduleResolve (node:internal/modules/esm/resolve:945:10)
    at defaultResolve (node:internal/modules/esm/resolve:1153:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:842:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

After hardcoding the route to the run.js script and invoking it via `node path/to/apps/content-manager-pro/extensions/layouts/directus-extension-layout-revisions/node_modules/@directus/extensions-sdk/dist/cli/run.js build`.

```sh
directus-extension-layout-revisions@1.0.0 build
> node /path/to/apps/content-manager-pro/extensions/layouts/directus-extension-layout-revisions/node_modules/@directus/extensions-sdk/dist/cli/run.js build

node:fs:600
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open '/path/to/apps/content-manager-pro/extensions/layouts/directus-extension-layout-revisions/node_modules/@directus/package.json'
    at Object.openSync (node:fs:600:3)
    at readFileSync (node:fs:468:35)
    at file:///path/to/apps/content-manager-pro/extensions/layouts/directus-extension-layout-revisions/node_modules/@directus/extensions-sdk/dist/cli/utils/get-sdk-version.js:5:24
    at ModuleJob.run (node:internal/modules/esm/module_job:194:25) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/path/to/apps/content-manager-pro/extensions/layouts/directus-extension-layout-revisions/node_modules/@directus/package.json'
}

Node.js v18.13.0
```

--- a/packages/create-directus-extension/lib/index.js
+++ b/packages/create-directus-extension/lib/index.js
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 'use strict';
 
-import { inquirer } from 'inquirer';
+import inquirer from 'inquirer';
 import { EXTENSION_LANGUAGES, EXTENSION_TYPES, BUNDLE_EXTENSION_TYPES } from '@directus/constants';
 import { create } from '@directus/extensions-sdk/cli';
 

--- a/packages/extensions-sdk/src/cli/utils/get-template-path.ts
+++ b/packages/extensions-sdk/src/cli/utils/get-template-path.ts
@@ -1,5 +1,6 @@
-import path from 'path';
+import { dirname, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
 
 export default function getTemplatePath(): string {
-	return path.resolve(__dirname, '..', '..', '..', '..', 'templates');
+	return resolve(dirname(fileURLToPath(import.meta.url)), '..', '..', '..', '..', 'templates');
 }

--- a/packages/extensions-sdk/src/cli/utils/get-template-path.ts
+++ b/packages/extensions-sdk/src/cli/utils/get-template-path.ts
@@ -2,5 +2,5 @@ import { dirname, resolve } from 'node:path';
 import { fileURLToPath } from 'node:url';
 
 export default function getTemplatePath(): string {
-	return resolve(dirname(fileURLToPath(import.meta.url)), '..', '..', '..', '..', 'templates');
+	return resolve(dirname(fileURLToPath(import.meta.url)), '..', '..', '..', 'templates');
 }

